### PR TITLE
Clean up code and small refactor

### PR DESCRIPTION
**Pull request by KhoraLee**
Removed redundant `self`, `return`.
Removed types where they can be easily inferred. ex) `var tmp: String = ""` -> `var tmp = ""`
Replaced tab with 4 spaces.
Fix wrong indentation.
Wrap function call more readable.
And so on...

### CODE REVIEW

**Comment by KhoraLee** [comment]
This will also remove unused variable in `AppSettings.swift` which I forgot to remove in #266. (`var keymap: [[CGFloat]] = []`)

File: PlayCover/Utils/PlayTools.swift [hunk 1]
```diff
@@ -120,44 +121,41 @@ class PlayTools {
...
     private static var install_name_tool: URL {
-		get throws {
-			try binPath("install_name_tool")
-		}
+        get throws {
+            try binPath("install_name_tool")
+        }
     }
 
     private static var ldid: URL {
```
Comment (Depal1) [inline PlayCover/Utils/PlayTools.swift:150]: I thought we no longer needed ldid. Perhaps this is dead code.

**Comment by IsaacMarovitz** [comment]
> Replaced tab with 4 spaces.

Not sure about this

**Comment by KhoraLee** [comment]
> > Replaced tab with 4 spaces.
> 
> Not sure about this

Xcode's default indent behavior is 4 spaces. And most of code base is already based on 4 space.

**Comment by Depal1** [comment]
I am going to merge, and leave the ldid matter for later.

**Pull request merged**